index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li class="nav-logo"><a href="#">My Logo</a></li>
+                <li><img src="images/Menu.svg" alt="menu image" class="menu-image"></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="flex-container">
+    <section class="greet-section"> 
+      
+        <h1>I’m Mariane Glad to see you!</h1>
+    
+        <p class="info-section">I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    
+        
+        
+    </section>
+    <header class="social">
+        <p class="connect">LET’S CONNECT</p>
+        <ul>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectortweet.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Linkedin.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectorm.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectorgit.svg" alt="social media image"></a></li>
+            <li><a href="https://www.linkedin.com/in/rhyine-stewart-a8373913a/"><img src="images/Vectorpeace.svg" alt="social media image"></a></li>
+        </ul>
+        
+    </header>
```

Comment:
- [ ] OPTIONAL: I don't think more than one header on a page is a good HTML practice. You may consider a div-tag here and better wrap it in the section element. :pray: